CONTRIBUTING.md

```diff
@@ -33,6 +33,21 @@ Here is an overview of [how `@octokit/rest` works](HOW_IT_WORKS.md).
 * Be sure to tag any issues your pull request is taking care of / contributing to.
 	* Adding "Closes #123" to a pull request description will auto close the issue once the pull request is merged in.
 
+## Testing a pull request from github repo locally:
+
+* You can install a specific pull request of a module from github using the bellow command
+`npm install owner/reponame#pullrequestId/branchname`
```

Comment:
oh sorry I’ve only seen that now, can you please change `owner/reponame` to `octokit/rest.js` and `pullrequestId/branchname` to just `branchname`? I just happen to prefix my branchnames with the issue number that they are addressing.

Reply:
I felt this as explaining a general command to use irrespective of `octokit` and the below line will take the case of octokit and show it as an example. Let me know if you think its going to be confusing rather than helpful! Will change that! 🙂 

> pullrequestId/branchname to just branchname? I just happen to prefix my branchnames with the issue number that they are addressing.

ah..! Gotcha! Will do that!